tests/e2e/tests/pilot/testdata/v1alpha1/authn-policy-jwt.yaml.tmpl

```diff
@@ -0,0 +1,15 @@
+apiVersion: "authentication.istio.io/v1alpha1"
```

Comment:
everything under v1alpha1 is considered as old routing code that will end up being deleted.
So I suggest putting the authN stuff under testdata/authn/v1alpha1/